clis/teliod/src/config.rs

```diff
@@ -50,9 +52,21 @@ fn reconnect_after_expiry_default() -> Percentage {
     Percentage(90)
 }
 
-#[derive(PartialEq, Eq, Deserialize, Debug)]
+#[derive(Deserialize, Debug, Default)]
+pub struct ClientConfig {
+    pub hw_identifier: String,
+    pub auth_token: String,
+    pub private_key: SecretKey,
+    pub public_key: PublicKey,
```

Comment:
If public key is derived from private key, why do we need to provide both? Maybe we could use libtelio `generate_public_key(secret_key: SecretKey)` function.
What do you think? this could prevent the user accidentally giving a wrong pair?

Reply:
PublicKey is generated from PrivateKey. We don't save the keypair.

Reply:
Removed from user config and made part of a separate file to be read/written but not user defined.